src/coreclr/jit/lowerxarch.cpp

```diff
@@ -3860,7 +3857,8 @@ GenTree* Lowering::LowerHWIntrinsicGetElement(GenTreeHWIntrinsic* node)
             new (comp, GT_LEA) GenTreeAddrMode(addr->TypeGet(), newBase, newIndex, newScale, newOffset);
         BlockRange().InsertBefore(node, newAddr);
 
-        GenTreeIndir* newIndir = comp->gtNewIndir(newType, newAddr, (indir->gtFlags & GTF_IND_FLAGS));
+        GenTreeIndir* newIndir =
```

Comment:
Would it be good to cache the `JITtype2varType(simdBaseJitType)` somewhere, rather than duplicating it in 3 places?

Reply:
I preferred the minor duplication here since we don't expect these transformations to kick in that often, so we would usually end up not using the computed value if we did it before.